src/store/ChartState.ts

```diff
@@ -510,6 +514,9 @@ class ChartState {
         if (this.context && isTimeUnitSecond && isChartTypeCandle) {
             this.setChartType('mountain'); // if granularity is zero, set the chartType to 'mountain'
         }
+        if (granularity === 0 && this.mainStore.studies.hasPredictionIndicator) {
+            this.mainStore.studies.deletePredictionStudies();
```

Comment:
Helps clear those indicators that are inapplicable for a chart mode that we are switching to.